Source/WebCore/animation/KeyframeInterpolation.cpp

```diff
@@ -121,7 +121,14 @@ const KeyframeInterpolation::KeyframeInterval KeyframeInterpolation::interpolati
     return { intervalEndpoints, hasImplicitZeroKeyframe, hasImplicitOneKeyframe };
 }
 
-void KeyframeInterpolation::interpolateKeyframes(Property property, const KeyframeInterval& interval, double iterationProgress, double currentIteration, Seconds iterationDuration, TimingFunction::Before before, const CompositionCallback& compositionCallback, const AccumulationCallback& accumulationCallback, const InterpolationCallback& interpolationCallback, const RequiresBlendingForAccumulativeIterationCallback& requiresBlendingForAccumulativeIterationCallback) const
+static double transformProgressDuration(const CSSNumberishTime& duration)
```

Comment:
I don't really understand what this function is used for.

Reply:
Yeah… this is specific to the `CubicBezierFunction` and  type of `TimingFunction::transformProgress()` for the `UnitBezier` solver. I suppose we could pass down the `CSSNumberishTime` though here.